### ISSUE TITLE
Exception: File output\pretrain\DarkNet53 is not a paddle parameter file

### ISSUE THREAD

**Issue by mzxhzhp** [PaddlePaddle/PaddleX#850]
问题类型：模型训练  
**问题描述**  

====================  
请在这里描述您在使用过程中的问题，如模型训练出错，建议贴上模型训练代码，以便开发人员分析，并快速响应

在Windows平台，Paddle 版本2.1.0，使用YOLOv3进行模型训练时报错：Exception: File output\pretrain\DarkNet53 is not a paddle parameter file。
具体代码如下：
# 模型开始训练
num_classes = len(train_dataset.labels)
print("num_classes = %d" % num_classes)
model = pdx.det.YOLOv3(num_classes=num_classes, backbone='DarkNet53')
model.train(
    num_epochs=40,
    train_dataset=train_dataset,
    train_batch_size=2,
    eval_dataset=eval_dataset,
    learning_rate=0.000125,
    lr_decay_epochs=[210, 240],
    # pretrain_weights=None,
    save_interval_epochs=10,
    save_dir='output',
    use_vdl=True)

我看了下，output\pretrain\DarkNet53 是从PaddleHub Server下载的预训练模型，不知道为啥会报错。奇怪的在AIStudio上没有这个报错。

**Comment by will-jl944** [comment]
请问你使用的PaddleX版本是多少呢？

**Comment by mzxhzhp** [comment]
paddle2onnx      0.6
paddlehub        2.1.0
paddlenlp        2.0.1
paddlepaddle     2.1.0
paddlepaddle-gpu 2.1.0
paddleslim       1.1.1
paddlex          1.3.10

**Comment by will-jl944** [comment]
你好，我们已经复现出了这个问题，建议升级到PaddleX 1.3.11版本，可使用指令`pip install -U paddlex==1.3.11`进行升级。

**Comment by mzxhzhp** [comment]
paddlex 升级到 1.3.11 后，问题解决了。
感谢解答。
